### PR TITLE
the one that makes buttons stick to their naming convention rather than 'fake it'

### PR DESCRIPTION
**Pull request by sturobson**
Currently the 'secondary' button is actually the primary button with an outline.

This updates the variants to stick the naming convention of what we call them to be the same in the code. 

Will match up with badges PR too

### CODE REVIEW

File: components/vf-button/CHANGELOG.md
Comment (khawkins98) [inline components/vf-button/CHANGELOG.md:3]: I think like the vf-badge pull request, this changelog would benefit from a mini "how to migrate" note.

Comment (sturobson) [reply components/vf-button/CHANGELOG.md:3]: > a mini "how to migrate"

if the usage of existing primary, secondary, tertiary badges and buttons were implemented 'semantically' then I don't think there would be a need. 

So, yes - you're probably right -- but there probably needs more documentation as to when to use what where and why.

Comment (sturobson) [reply components/vf-button/CHANGELOG.md:3]: @khawkins98 I've added the relevant migration details - I'll let you resolve this conversation and I will merge the PR once approved.

**Review by khawkins98 — APPROVED** [review]

File: components/vf-button/CHANGELOG.md
Comment (khawkins98) [reply components/vf-button/CHANGELOG.md:3]: 👍

**Pull request merged**
